dlt/common/libs/pyarrow.py

```diff
@@ -866,13 +867,17 @@ def convert_numpy_to_arrow(
                 try:
                     # the 3 types match those supported by `map_nested_in_place()`
                     if isinstance(value, (tuple, dict, list)):
-                        encoded_value = map_nested_in_place(custom_encode, value)
+                        encoded_value = map_nested_in_place(
```

Comment:
if we go with global var this won't be needed

Reply:
changed in 659cc5c